src/components/homePage.tsx

```diff
@@ -17,64 +27,142 @@ const CardTitle = ({ title, icon }: { title: string; icon: any }) => {
   );
 };
 
-const OPMCards = ({ value, time }: { value: string; time: string }) => {
+const OPMCards = ({ value }: { value: number }) => {
   return (
     <div className="flex items-end">
       <span className="text-2xl text-[#F2F2F2]">{value}</span>
-      <div className="border rounded-lg border-[#383F47] text-white text-[10px] h-fit p-1 ml-3">
-        Last {time} min
-      </div>
     </div>
   );
 };
 
-const ComparisonCards = () => {
+const ComparisonCards = ({
+  today,
+  lastDay,
+}: {
+  today: number;
+  lastDay: number;
+}) => {
+  const difference = lastDay - today;
   return (
     <div className="flex">
       <div className="flex flex-col pr-1 sm:pr-2 justify-between">
         <span className="text-[10px]">Today</span>
-        <span className="text-[#F2F2F2] text-2xl">38</span>
+        <span className="text-[#F2F2F2] text-xl">{today}</span>
       </div>
       <div className="border border-r border-[#383F47] h-[2.5rem] m-auto"></div>
       <div className="flex flex-col px-1 sm:px-2 justify-between">
         <span className="text-[10px]">Last Day</span>
-        <span className="text-[#F2F2F2] text-2xl">38</span>
+        <span className="text-[#F2F2F2] text-xl">{lastDay}</span>
       </div>
       <div className="border border-r border-[#383F47] h-[2.5rem] m-auto"></div>
       <div className="flex flex-col justify-between pl-1 sm:pl-2">
-        <span className="text-[10px] text-[#F16476]">Difference</span>
+        <span
+          className={`${
+            difference > 0
+              ? "text-[#F16476]"
+              : difference < 0
+              ? "text-[#5CB7ED]"
+              : "text-[#8B8C8F]"
+          } text-[10px]`}
+        >
+          Difference
+        </span>
         <div className="flex">
-          <span className="text-[#F2F2F2] text-[#F16476] text-2xl">38</span>
-          <CustomImage className="flex ml-2" src={trending_down} />
+          <span
+            className={`${
+              difference > 0
+                ? "text-[#F16476]"
+                : difference < 0
+                ? "text-[#5CB7ED]"
+                : "text-[#F2F2F2]"
+            } text-[#F16476] text-2xl`}
+          >
+            {!isNaN(difference) && difference}
```

Comment:
`NaN` is a falsy value in js.
So you can just do `{difference && difference}`

Reply:
this fails if value of difference is 0